packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Host, h } from '@stencil/core';
+import vaSeal from '../../assets/va-logo-white.svg';
+
+/**
+ * @componentName Minimal Footer
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-minimal-footer',
+  styleUrl: 'va-minimal-footer.scss',
+  shadow: true,
+})
+export class VAMinimalFooter {
+
+  render() {
+
+    return (
+      <Host>
+        <div class="va-footer">
+          <a href="/" >
+            <img class="va-logo" src={vaSeal} aria-hidden="true"/>
```

Comment:
I think because this is a functional image, and serves as the only content of the anchor tag, instead of `aria-hidden="true"` it needs some alt text to indicate that it's a link that takes the user to the va.gov home page. Something like `alt="VA.gov home"` 

More info here - https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo. 

And actually, I think the minimal header needs to be fixed as well - https://github.com/department-of-veterans-affairs/component-library/blob/f26ebe31dc5c4adfd445334bf816e8e97ac0e370/packages/web-components/src/components/va-minimal-header/va-minimal-header.tsx#L30

Reply:
@micahchiang thanks for the heads up. Also, [here is a PR](https://github.com/department-of-veterans-affairs/component-library/pull/938) to update minimal header.

Reply:
I left a [comment on the other PR](https://github.com/department-of-veterans-affairs/component-library/pull/938#discussion_r1385253503) and I just wanted to mention here that we can probably use the existing alt text from the current VA.gov footer logo: "VA logo and Seal, U.S. Department of Veterans Affairs"

There is also a `title` on the link "Go to VA.gov":

![Screenshot 2023-11-07 at 2 44 47 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/bdef9fad-d102-46d2-a2b5-ffcbeb5b30b2)